README.md

```diff
@@ -1 +1,63 @@
-# Stock-market
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# stock-market
+
+> This is a SPA that shows financial statements in real time consuming [Financial modeling prep API](https://financialmodelingprep.com/developer/docs/)
+
+
+![screenshot desktop]('https://github.com/franciscoPonceDev/stock-market/blob/develop/src/assets/fierce-escarpment-98670.herokuapp.com_(Desktop).png')
+
+![screenshot mobile]('https://github.com/franciscoPonceDev/stock-market/blob/develop/src/assets/fierce-escarpment-98670.herokuapp.com_(iPhone%206_7_8).png')
+
+
+## Built With
+
+- React
+- Redux
+- Sass
+
+## Live Demo
+
+[Live Demo Link](https://fierce-escarpment-98670.herokuapp.com)
+
+## Video Link
+
+[Project video Link](https://www.loom.com/share/3ee1aa10eb1443308c6203a671101883)
+
+## Getting Started
+
+### To get a local copy up and running follow these simple example steps.
+
+- Clone the repo with `git@github.com:franciscoPonceDev/stock-market.git`
+- Open your terminal or command line
+- Run `git clone` and paste the link
+- Open the folder with your code editor
+- Create a branch to work on
+- Run `npm install`
+- Run `npm start` to launch the server.
+
+## Author
+
+👤 **Francisco Ponce**
+
+## Francisco's Profile
+
+- GitHub: [@franciscoPonceDev](https://github.com/franciscoPonceDev)
+- Twitter: [@franciscoPonce](https://twitter.com/franciscoPonce)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/dev-ponce/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/franciscoPonceDev/stock-market/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+- A special thanks to Microverse for inspiring this project. We are indebted to you all at Microverse
+- A special thanks goes to all our peers at Microverse for being there for us.
```

Comment:
- [ ] The Licence section of your README file from this template is missing. Please let's add it. This will render our README file a professional one.